src/components/EpisodesNav/styles.scss

```diff
@@ -1,3 +1,14 @@
 .paginationLink {
 	font-size: 1em; //placeholder to have the classname showing in inspector
+}
+
+.navButton {
+	width: 50px;	//placeholder so they are visible
+	height: 50px;
+	margin: 5px;
+
+	&[aria-hidden=true] {
```

Comment:
does this work? I think you need `&[aria-hidden="true"]`. I might be wrong, though.

Reply:
It works, but as discussed not best practice.